packages/experimental/Button/src/ButtonTokens.ts

```diff
@@ -11,10 +11,9 @@ export const defaultButtonTokens: TokenSettings<ButtonTokens, Theme> = () =>
     medium: {
       padding: globalTokens.spacing.sNudge - globalTokens.stroke.width.thin,
       borderWidth: globalTokens.stroke.width.thin,
-      iconSize: 16,
+      iconSize: 20,
```

Comment:
The 16 is correct, this should be reverted. See my comment on #1237

Reply:
Same with iconSize for small and large

Reply:
Thank you for catching it :) Fixed